sites/partners/src/pages/index.tsx

```diff
@@ -174,11 +203,24 @@ export default function ListingsList() {
             headerContent={
               <div className="flex-row">
                 {isAdmin && (
-                  <LocalizedLink href={`/listings/add`}>
-                    <Button className="mx-1" onClick={() => false}>
-                      {t("listings.addListing")}
+                  <div className="flex-row">
+                    <LocalizedLink href={`/listings/add`}>
```

Comment:
I don't know if I'm looking at the right figma doc. But looks like between this, the user export, and the design we have slightly different looks of the buttons. The most prominent is the size of the buttons and first button not being white.
This:
![image](https://user-images.githubusercontent.com/42942267/225725846-33a1ce30-8393-40b9-8d88-dd5e1622fd63.png)

User export:
![image](https://user-images.githubusercontent.com/42942267/225725960-7fd6dea0-df45-446e-9085-f15aaf5d8452.png)

Design:
![image](https://user-images.githubusercontent.com/42942267/225726052-4956965f-e49c-427f-ab0f-462f5b139d2f.png)
I'm curious if we have a final design pattern that we should be doing. Either updating it here or on the user export

Reply:
Good catch! The button color was just a miss on my part so that should be aligned with the Figma now. In terms of the button sizing, the button padding in both the Figmas (units and listings) aligns with the base button size as opposed to the small one used in the User PR. If it works for you, I can align the user button size with what is currently on listings. Or I could touch base with design if you'd prefer that @ludtkemorgan ?

Reply:
I see. I think my confusion was around the button being bigger than the search bar since the designs have both of them the same height. I think it makes sense to keep it bigger and update the user button. Then we can get input from design on the search bar. Sorry for the confusion, I should have noticed it my user export